app/models/sender.rb

```diff
@@ -6,7 +6,7 @@ class Sender < ApplicationRecord
 
   validates :email, presence: true, format: { with: /\A.+\@.+\.[a-z]{1,4}\z/}
   validates :recipient, presence: true, format: { with: /\A.+\@.+\.[a-z]{1,4}\z/}
-  validates :file, presence: true, file_size: { less_than: 200.megabytes,
-    message: "must not exceed 200Mo. Please choose a smaller file." }
+  # validates :file, presence: true, file_size: { less_than: 200.megabytes,
+    # message: "must not exceed 200Mo. Please choose a smaller file." }
   validates_with MyValidator
```

Comment:
@prevostjules Please rename this

Reply:
@ManyBaby : Fait, MyValidator renommé en MailUniqueness

Reply:
My bad j'ai laissé un nom pas terrible